.github/workflows/test-unit.yml

```diff
@@ -33,22 +33,24 @@ jobs:
           go-version: '1.13.15'
       - uses: actions/checkout@v3
         with:
-            fetch-depth: 2
+          fetch-depth: 2
       - uses: actions/setup-node@v3
         timeout-minutes: 5 # See https://github.com/actions/cache/issues/810
         with:
-            node-version: ${{ matrix.node }}
+          node-version: ${{ matrix.node }}
```

Comment:
@styfle this spacing fix seems correct. If anything we probably need to be enforcing format on these files

Reply:
I would prefer the spacing is fixed in a different PR if possible and added to prettier (since it seems like its currently not auto-formatted)

Reply:
Fair enough - ill revert and make sure to follow up